File: samples/python/01_chat_openai.py
"""
01_chat_openai.py — Basic Chat Completions against OpenAI.
Run:
    pip install -r requirements.txt
    export OPENAI_API_KEY=sk-...
    python 01_chat_openai.py
"""
import os
from dotenv import load_dotenv
from openai import OpenAI

load_dotenv()

client = OpenAI(api_key=os.environ["OPENAI_API_KEY"])

resp = client.chat.completions.create(
    model="gpt-5.5",
    messages=[
        {"role": "system", "content": "You are concise."},
        {"role": "user", "content": "What is the capital of France?"},
    ],
)
print(resp.choices[0].message.content)

File: samples/typescript/01_chat_openai.ts
// 01_chat_openai.ts — Basic Chat Completions against OpenAI.
import "dotenv/config";
import OpenAI from "openai";

const client = new OpenAI({ apiKey: process.env.OPENAI_API_KEY });

const resp = await client.chat.completions.create({
  model: "gpt-5.5",
  messages: [
    { role: "system", content: "You are concise." },
    { role: "user", content: "What is the capital of France?" },
  ],
});
console.log(resp.choices[0].message.content);
